lessons/pydata/profiling/index.ipynb

```diff
@@ -0,0 +1,129 @@
+{
```

Comment:
<p>&gt; který je dostupný na&nbsp;<a href="https://github.com/PyDataCZ/naucse.python.cz/raw/a4baddab8b6608cfad2f5495200ac1d3fe78e66f/lessons/pydata/profiling/static/spotify_report.html" target="_blank">této stránce</a>.</p><p> </p><p>Mně to posílá s MIME <strong>Content-Type: </strong>text/plain; charset=utf-8, což může být pro nezvyklého trochu matoucí. Nechceš popsat, že je potřeba to stáhnout a prohlížet offline?</p><p> </p><p>&gt; Proč se tak stalo se dozvíme</p><p> </p><p>Proč se tak stalo, se...</p><p> </p><p>&gt; ...a o kompozici jednotlivých hodnot</p><p> </p><p>...a dalších vlastnostech popisující souhrnně vyskytující se hodnoty ("Composition").    [on je ten název divný už v angličtině, v češtině nedává smysl vůbec]</p><p> </p><p> &gt; Třetí část je věnovaná grafické reprezentaci různých korelačních koeficientů.</p><p> </p><p>Třetí část graficky reprezentuje několik různých korelačních koeficientů mezi páry proměnných.</p><p> </p><p>&gt; Poslední část reportu obsahuje prvních pět a posledních pět řádku datasetu k nahlédnutí.</p><p> </p><p>...deset... (aspoň v tom souboru, co jsem si stáhl)</p><p> </p><p>&gt; Pokud se detekce na první pokus nepodaří přichází na řadu úprava</p><p> </p><p>Pokud se detekce na první pokus nepodaří, přichází...</p><p> </p><p>&gt; &nbsp;Ne vše je ale v reportu obsaženo a to co tam je</p><p> </p><p>&nbsp;Ne vše je ale v reportu obsaženo[, optional] a to, co</p><p> </p><p>&gt; obsahují&nbsp;<a href="https://github.com/pandas-profiling/pandas-profiling#examples" target="_blank">stránky prokejtu</a>&nbsp;řadu příkladů reportů z různých datasetů.</p><p> </p><p>...proJEKtu... ukázkových reportů z různých datasetů.</p><p> </p><p>&gt; co ty ne resp.&nbsp;</p><p> </p><p>co ty ne, resp.&nbsp;</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/PyDataCZ/naucse.python.cz/pull/34/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='5'/>

Reply:
Jo, takhle to může být. Přidávám jen pár drobných návrhů na jazykové úpravy.